website/tests/acceptance/a11y-test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable qunit/no-only */
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import {
+  visit,
+  setupContext,
+  setupApplicationContext,
+  teardownContext,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'website/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+import { getOwner } from '@ember/application';
+
+module.only('Acceptance | a11y tests', function (hooks) {
+  setupApplicationTest(hooks);
+
+  let list = [];
+
+  // the idea here is to boot the app and pull out the list of routes to visit
+  // this does work, however the .forEach() runs first so this doesn't actually work work
+  hooks.before(async function () {
+    await setupContext(this);
+    await setupApplicationContext(this);
+    await visit('/');
+    // debugger;
+    const owner = getOwner(this);
+    const app = owner.lookup('controller:application');
+    list = app.get('model.toc.flat').map((page) => page.pageURL);
+    await teardownContext(this);
+  });
+
+  // a hard coded list does work...
+  list = ['components/badge', 'components/button'];
```

Comment:
i don't think such a list exists anywhere, but if we _could_ generate it somehow we could read it in here

Reply:
can we read it from the JSON generated at build time? similar to what we do here: https://github.com/hashicorp/design-system/blob/main/website/lib/markdown/index.js#L65-L82

Reply:
i'm not sure. i tried that but ran into issues doing node things in the context of how these tests are run.

Reply:
it could run into the same issue though in that the array needs to be defined _before_ the tests start trying to run